### PR TITLE
debug rules #2764

### PR DESCRIPTION
**Pull request by phpgeekfr**

### CODE REVIEW

File: firestore.rules [hunk 1]
```diff
@@ -133,15 +133,31 @@ service cloud.firestore {
 
     /// FUNCTIONS ///
 
+    function isAllowedToReadInvitation() {
+      return (
+        ( existingData().type == 'attendEvent' && isSignedIn() )
+        || isConcerned()
+      )
+    }
+
```
Comment (GrandSchtroumpf) [inline firestore.rules:142]: Add a comment : `// is host or guest`

File: libs/notification/src/lib/invitation/invitation-list/invitation-list.component.ts [hunk 2]
```diff
@@ -1,11 +1,8 @@
...
-import { InvitationQuery, InvitationService, InvitationStore, Invitation } from '../+state';
-import { Observable, Subscription, combineLatest } from 'rxjs';
-import { AuthQuery } from '@blockframes/auth/+state/auth.query';
+import { InvitationQuery, Invitation } from '../+state';
+import { Observable, Subscription } from 'rxjs';
 import { DateGroup } from '@blockframes/utils/helpers';
 import { ThemeService } from '@blockframes/ui/theme';
-import { PermissionsQuery } from '@blockframes/permissions/+state/permissions.query';
-
 @Component({
```
Comment (GrandSchtroumpf) [inline libs/notification/src/lib/invitation/invitation-list/invitation-list.component.ts:6]: Add a comment to say that this component will be removed after homogenisation of festival & catalog.

File: libs/user/src/lib/auth/pages/identity/identity.component.ts [hunk 3]
```diff
@@ -54,7 +54,9 @@ export class IdentityComponent {
       });
 
       // Accept the invitation from the organization.
-      const invitations = await this.invitationService.getValue(ref => ref.where('toUser.uid', '==', this.query.userId));
+      const invitations = await this.invitationService.getValue(ref => ref.where('mode', '==', 'invitation')
+        .where('type', '==', 'joinOrganization')
+        .where('toUser.uid', '==', this.query.userId));
```
Comment (GrandSchtroumpf) [inline libs/user/src/lib/auth/pages/identity/identity.component.ts:59]: You have a parenthesis too much I think.

Comment (phpgeekfr) [reply libs/user/src/lib/auth/pages/identity/identity.component.ts:59]: nop

**Pull request merged**